### PR TITLE
Added userParameterString to trajectoryParams

### PR DESCRIPTION
**Pull request by dchansen**

### CODE REVIEW

File: matlab/+ismrmrd/AcquisitionHeader.m [hunk 1]
```diff
@@ -70,6 +70,10 @@
             'ACQ_IS_DUMMYSCAN_DATA',                   27, ...
             'ACQ_IS_RTFEEDBACK_DATA',                  28, ...
             'ACQ_IS_SURFACECOILCORRECTIONSCAN_DATA',   29, ...
+            'ACQ_COMPRESSION1',                        53, ...
```
Comment (hansenms) [inline matlab/+ismrmrd/AcquisitionHeader.m:73]: This is covered in #212

File: CMakeLists.txt [hunk 2]
```diff
@@ -85,7 +85,7 @@ set(ISMRMRD_VERSION_PATCH 0)
 set(ISMRMRD_VERSION_STRING ${ISMRMRD_VERSION_MAJOR}.${ISMRMRD_VERSION_MINOR}.${ISMRMRD_VERSION_PATCH})
 set(ISMRMRD_SOVERSION ${ISMRMRD_VERSION_MAJOR}.${ISMRMRD_VERSION_MINOR})
 
-set(ISMRMRD_XML_SCHEMA_SHA1 "22137b8bcbe7fe670019c61d3c645b6c9289641a")
+set(ISMRMRD_XML_SCHEMA_SHA1 "37a9f39f5d5e615ea67919064aef269938812519")
```
Comment (hansenms) [inline CMakeLists.txt:88]: You should bump the version when you update this.

**Review by hansenms — CHANGES_REQUESTED** [review]
Couple of minor comments.

File: CMakeLists.txt (hunk 2 above)
Comment (dchansen) [reply CMakeLists.txt:88]: Fixed

File: matlab/+ismrmrd/AcquisitionHeader.m (hunk 1 above)
Comment (dchansen) [reply matlab/+ismrmrd/AcquisitionHeader.m]: No longer part of the changeset

**Review by hansenms — APPROVED** [review]

**Pull request merged**